bcgov/HOUS-permit-portal
# Bphh 2479 compliance HVAC

## Description
- Add Stepcode Checklist HVAC compliance section


## What type of PR is this? (check all applicable)


## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-2479


## Steps to QA
- Create any permit application

- Append to permit application edit route part-3-step-code

- Select HVAC from side bar

- Fill up the radio select form. A text field should also show up if "Other" is chosen for any of the fields

![Screenshot 2024-12-12 at 3 41 54 PM](https://github.com/user-attachments/assets/7bb61320-494e-4f1e-a1b1-7ce7c1481a72)
